scripts/import/laptops/update_visit_data

```diff
@@ -143,9 +146,34 @@ forms_date_increments = {
 }
 
 # First REDCap connection for the Summary project (this is where we put data)
-summary_key_file = open(os.path.join(os.path.expanduser("~"), '.server_config/redcap-dataentry-token'), 'r')
-summary_api_key = summary_key_file.read().strip()
-rc_summary = redcap.Project('https://ncanda.sri.com/redcap/api/', summary_api_key, verify_ssl=False)
+def connect_to_redcap():
+    summary_key_file = open(os.path.join( os.path.expanduser("~"), '.server_config/redcap-dataentry-token'), 'r')
+    summary_api_key = summary_key_file.read().strip()
+    rc = redcap.Project(redcap_server, summary_api_key, verify_ssl=False)
+    return rc
+
+# If connection to redcap server fail, try multiple times
+errLoop = 0
```

Comment:
I thought we are moving all this logic into the sibis package. Why add all the retrying code to each file if the next thing we plan on doing is removing it?